packages/kernel-6.12/neuron-resolve-static-const-compiler-warnings.patch

```diff
@@ -0,0 +1,151 @@
+From 5d1dae633d16144e6fe21da2e70b179a0c009da3 Mon Sep 17 00:00:00 2001
```

Comment:
Is there a reason we haven't named this patch prefixed with `1008-` ?

Reply:
Yes: this patch isn't applied when we autopatch the world with the 1000-series patches. It gets applied explicitly as part of unpacking and building the neuron driver.